scratch/evaluate_qd_channel_lroom_scenario.cc

```diff
@@ -550,10 +555,10 @@ main (int argc, char *argv[])
               std::cout << "Flow " << i->first << " (" << t.sourceAddress << " -> " << t.destinationAddress << ")" << std::endl;
               std::cout << "  Tx Packets: " << i->second.txPackets << std::endl;
               std::cout << "  Tx Bytes:   " << i->second.txBytes << std::endl;
-              std::cout << "  TxOffered:  " << i->second.txBytes * 8.0 / ((simulationTime - 1) * 1e6)  << " Mbps" << std::endl;
+              std::cout << "  TxOffered:  " << i->second.txBytes * 8.0 / ((simulationTime - appStartTime.GetSeconds ()) * 1e6)  << " Mbps" << std::endl;
```

Comment:
repeated non-trivial command: save it as a variable to make it a single point of failure

Reply:
What's the repeated command? The simulation duration computation?